server.go

```diff
@@ -14,20 +14,31 @@ import (
 )
 
 type Server struct {
-	Port         int
-	ProxyTimeout time.Duration
-	routes       []Route
-	Target       string
-	Logger       *log.Logger
-	httpServer   *http.Server
+	Port             int
+	ProxyTimeout     time.Duration
+	routes           []Route
+	Target           string
+	Logger           *log.Logger
+	httpServer       *http.Server
 	DefaultPageTitle string
+	ignoreHeaders    map[string]struct{}
 }
 
+var setMember struct{}
+
 func (s *Server) Get(path string, layout string, fragments []string) {
 	route := newRoute(path, layout, fragments)
 	s.routes = append(s.routes, *route)
 }
 
+func (s *Server) IgnoreHeader(name string) {
+	if s.ignoreHeaders == nil {
```

Comment:
This isn't great but works for now. We should probably add a `NewServer` function to set defaults when constructing a new server instance.